code-dot-org/code-dot-org
# Show other eligibility criteria to all teachers regardless of school eligibility

[PLC-742](https://codedotorg.atlassian.net/browse/PLC-742)

- Makes discount code application criteria visible to all applicants, instead of only to those whose schools are eligible. Implemented to make life easier for internal folks who are corresponding with teachers directly and want teachers to be able to access information about their status.
- Adds the school eligibility check as one of the check/x steps in the "Eligibility Requirements" list -- redundant, but maybe more clear than not including it.

Before:

![image](https://user-images.githubusercontent.com/25372625/74201878-c91c6580-4c1f-11ea-84e7-f6818bb477eb.png)

After (the "Eligibility Requirements" checklist is already visible to teachers who met the school bar):

![image](https://user-images.githubusercontent.com/25372625/74293824-d35a6480-4cf0-11ea-8e88-d7b7e4a43316.png)

## Testing story

Added tests to:
- confirm that one of the eligibility checks (the Unit 6 check) renders if a teacher's school does not meet the school eligibility bar. Tried this test without the new code (failed), and with the code changes (passed).
- confirm that the "Submit" button does not appear if a teacher has met the first two "Other" eligibility criteria but not the school eligibility criteria.

## Reviewer Checklist:
